app/views/landing/index.slim

```diff
@@ -0,0 +1,19 @@
+doctype html
```

Comment:
html, head, and body tags should be defined in application.html.slim

Also, this name should be named index.html.slim